Resources/Prototypes/_NF/Entities/Mobs/NPCs/base_humanoid_hostile.yml

```diff
@@ -55,10 +86,15 @@
 #      task: SimpleRangedHostileCompound
     blackboard:
       NavClimb: !type:Bool # They use this option too much for my liking, but without this mobs usually get stuck on spawn in expeds
-        true
+        false
   - type: NpcFactionMember
     factions:
-      - SimpleNeutral
+      - SimpleHostile
+  - type: Carriable # Carrying system from nyanotrasen.
+  - type: MobPrice
+    price: 1500 # Kidnapping a living person and selling them for cred is a good move.
+    deathPenalty: 0.5 # However they really ought to be living and intact, otherwise they're worth 2x less.
```

Comment:
Real.

Reply:
Also you cannot even sell alive mobs unless they are sold on a ship, with the ship.

Reply:
You can sell mob in crit though: I did sell critted syndies for FUCs.

Reply:
Not sure about hard cash though, haven't tried. 1500 ain't worth the hussle.